lib/home/library/video_library.dart

```diff
@@ -90,15 +85,15 @@ class VideoCard extends StatelessWidget {
   }
 
   // Add a method to download the video
-  Future<void> downloadVideo(BuildContext context) async {
+  Future<void> downloadVideo() async {
```

Comment:
We can use `navigator` instead of `ScaffoldMessenger.of(context)`, to avoid needing the `BuildContext` here.